src/components/PatronRequestForm/PatronRequestForm.js

```diff
@@ -149,6 +160,7 @@ const PatronRequestForm = ({ autopopulate, copyrightTypes, enabledFields,
           <Field
             component={RadioButton}
             inline
+            onClick={resetTier}
```

Comment:
Turns out if you use `onChange` for a radio button it prevents the change from happening even if you don't call `preventDefault()`